lisa/analysis/load_tracking.py

```diff
@@ -87,6 +88,7 @@ def _df_either_event(self, events):
 
         raise RuntimeError("Trace is missing one of either events: {}".format(events))
 
+    @requires_one_event_of('sched_load_cfs_rq', 'sched_load_avg_cpu')
```

Comment:
You should remove the mention of those in the docstring, since that's now automagic.